quill-cassandra-core/src/test/cql/cassandra-schema.cql

```diff
@@ -0,0 +1,198 @@
+CREATE KEYSPACE IF NOT EXISTS quill_test
```

Comment:
Is this a new file? we don't need to have multiple copies of it since the database can be reused